src/AppBundle/Controller/ProfileController.php

```diff
@@ -1502,4 +1510,19 @@ public function editCustomisationAction(ProfileAssociation $profileAssociation,
             'formsExample' => $formsExampleViews
         ));
     }
+
+    /**
+     * @Route("/profile/{id}/makevisible", name="profile_make_visible", requirements={"id"="^([0-9]+)|(profileID){1}$"})
+     * @param Profile $profile
+     * @return JsonResponse
+     */
+    public function makeVisibleAction(Profile $profile)
```

Comment:
Sécuriser l'accès à cette méthode avec $this->denyAccessUnlessGranted()

Reply:
Done